recipes/rhapsody/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set version = "0.2.1" %}
+{% set name = "rhapsody" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/r/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d73e0af9f57fdb7e48bd7bab4f946fbeb0240f6d44cb47e0257a277df7051beb
+
+
+build:
+  number: 0
+  skip: true  # [win]
```

This looks like it could be noarch?
```suggestion
  noarch: python
```
(It's fine if the Windows CI then fails due to missing dependencies)

---

If not, the version constraint should be removed from `python` in `host` and `run` as there will be one build per Python version per platform.